addons/data-category-tweaks-v2/userscript.js

```diff
@@ -152,6 +152,7 @@ export default async function ({ addon, console, msg, safeMsg }) {
   // https://github.com/scratchfoundation/scratch-vm/blob/a0c11d6d8664a4f2d55632e70630d09ec6e9ae28/src/engine/runtime.js#L1381
   const originalGetBlocksXML = vm.runtime.getBlocksXML;
   vm.runtime.getBlocksXML = function (target) {
+    ScratchBlocks.Msg["CATEGORY_LISTS"] = msg("list-category");
```

Comment:
Is this name arbitrary? If only we are going to use it, why not choose a less general name?

Reply:
it follow the naming consistancy of scratch but is could be sa-* if wanted.

Reply:
@TheColaber Change it to `SA_CATEGORY_LISTS` then